README.md

```diff
@@ -138,6 +140,44 @@ func main() {
 }
 ```
 
+`go-githubauth` implements a set of `oauth2.TokenSource` to be used with `oauth2.Client`. An `oauth2.Client` can be injected into the `github.Client` to authenticate requests.
+
+Other example using `go-githubauth`:
+
+```go
+package main
+
+import (
+ "context"
+ "fmt"
+ "os"
+ "strconv"
+
+ "github.com/google/go-github/v62/github"
+ "github.com/jferrl/go-githubauth"
+ "golang.org/x/oauth2"
+)
+
+func main() {
+ privateKey := []byte(os.Getenv("GITHUB_APP_PRIVATE_KEY"))
+
+ appTokenSource, err := githubauth.NewApplicationTokenSource("app-id", privateKey)
+ if err != nil {
+  fmt.Println("Error creating application token source:", err)
+  return
+ }
+
+ installationTokenSource := githubauth.NewInstallationTokenSource(20, appTokenSource)
```

Comment:
According to this: https://github.com/jferrl/go-githubauth/blob/main/auth.go#L52
the first argument is `id string` so the `20` here looks weird.

Reply:
I've updated the code to be aligned with go-github ids representation. Now both ids are defined as **int64**. Also changed the readme go code examples.

Reply:
Just FYI - your README.md still needs to be updated - not a biggie - just wanted to let you know.  😄 
https://github.com/jferrl/go-githubauth/blob/main/README.md?plain=1#L48-L51

Reply:
Sure! Im working on it! thanks!